WordPress/gutenberg
# Use Panel for navigation structure on small screens only

## Description
The navigation structure uses the Panel component to make it possible for mobile clients to quickly skip it and scroll down. On larger screens, it is placed to the left of the menu preview on large screens and the usage of Panel does not make sense as much. This PR uses Card on larger screens and Panel on smaller ones.

## How has this been tested?
1. Enable the navigation experiment in Gutenberg > Experiments
1. Go to Gutenberg > Navigation (beta)
1. Confirm that on large screen it is not possible to collapse the navigation structure area.
1. Confirm that on small screen it is possible to collapse the navigation structure area.

## Screenshots 

**Small screen:**
<img width="704" alt="Zrzut ekranu 2020-06-2 o 15 13 33" src="https://user-images.githubusercontent.com/205419/83524832-71fb1300-a4e4-11ea-9464-4962032e2f42.png">

**Large screen:**
<img width="1174" alt="Zrzut ekranu 2020-06-2 o 15 13 20" src="https://user-images.githubusercontent.com/205419/83524825-70c9e600-a4e4-11ea-8346-ceb07d197a44.png">

## Types of changes
Non-breaking change

## Checklist:
